action.yml

```diff
@@ -73,7 +85,7 @@ runs:
         echo "binDir=${webotsBinDir}" >> $GITHUB_OUTPUT
         echo "bin=${webotsBin}" >> $GITHUB_OUTPUT
         echo "pkgName=$pkgName" >> $GITHUB_OUTPUT
-        echo "tag=$webotsTag" >> $GITHUB_OUTPUT
+        echo "baseUrl=$webotsBaseUrl" >> $GITHUB_OUTPUT
```

Comment:
The `tag` action output should be updated.

Reply:
I've replaced the tag output with baseUrl in my latest commit.